### PR TITLE
♻️  Cleaned up workflow

### PR DESCRIPTION
**Pull request by jjperezaguinaga**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Interface config
+HOST_IPV4=0.0.0.0:9091 // Default IPv4
```
Comment (peterbraden) [inline .env.example:2]: Do we have a default in the code? If this variable is omitted what do we use?

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -84,43 +80,38 @@ jobs:
...
 
   publish_chat_dist:
     name: Creates release with node binary per OS
-    if: startsWith(github.ref, 'refs/heads/releases/chat')
     runs-on: ubuntu-latest
     needs: [build_chat_binaries]
     steps:
       - name: Get HOPR Chat (linux)
-        uses: actions/download-artifact@master
+        uses: actions/download-artifact@v2
```
Comment (peterbraden) [inline .github/workflows/publish.yml:106]: Why not master here?

File: .env.example (hunk 1 above)
Comment (jjperezaguinaga) [reply .env.example:2]: Sadly no, we should tho for sure.

File: .github/workflows/publish.yml (hunk 2 above)
Comment (jjperezaguinaga) [reply .github/workflows/publish.yml:106]: `master` changes, whereas `v2` uses a fixed tag we know it's working.

Comment (peterbraden) [reply .github/workflows/publish.yml:106]: oh, I misread - this is the action version, I see.

**Pull request merged**